aws/signer/v4/v4_test.go

```diff
@@ -62,7 +62,7 @@ func TestStripExcessHeaders(t *testing.T) {
 }
 
 func buildRequest(serviceName, region, body string) (*http.Request, io.ReadSeeker) {
-	reader := strings.NewReader(body)
+	reader := &readerSeekerWrapper{strings.NewReader(body)}
```

Comment:
We may want to have a separate benchmarks and tests for this functionality and leave these benchmarks the same since we want to ensure non-`ReadSeeker` interfaces do not break.